src/main/java/org/sqlite/jdbc3/JDBC3DatabaseMetaData.java

```diff
@@ -1173,9 +1172,8 @@ public ResultSet getCrossReference(
     /** @see java.sql.DatabaseMetaData#getSchemas() */
```

Comment:
My understanding is that schemas are currently stored instead of being retrieved everytime, but as databases can be attached dynamically, we should probably get the schemas all the time, and add a test to cover that too.

Reply:
still not sure about this one, i see you resolved it, but as `getSchemas` is only initialized if it's null, it would not be updated after a database is attached or detached.